app/models/shipit/deploy.rb

```diff
@@ -39,6 +39,16 @@ def append_status(task_status)
 
     delegate :broadcast_update, :filter_deploy_envs, to: :stack
 
+    def self.newer_than(commit)
+      return all unless commit
+      where('id > ?', commit.try(:id) || commit)
```

Comment:
I suppose you copy pasted this from `Commit`, because `id` in this context is a `deploy.id`, so that argument should probably be `deploy`